_includes/getting-started-template.html

```diff
@@ -61,6 +61,38 @@ <h1 id="templates" class="page-header">Templates</h1>
 
 </div>
 
+<h3><a name="user-content-commonjs-support" class="anchor" href="#commonjs-support"><span class="octicon octicon-link"></span></a>CommonJS Support</h3>
+
+<p>If you use a CommonJS build system (such as <a href="https://webpack.github.io/">Webpack</a> or <a href="http://browserify.org/">Browserify</a>), we recommend only loading the controls you need. You can also load all the controls by including the npm package.</p>
+
+<h4>Working with Browserify</h4>
+
+<p>This should work out of the box with Browserify.</p>
+
+<h4>Working with webpack</h4>
+
+{% highlight js %}
+npm install --save-dev imports-loader
+// or
+yarn add --dev imports-loader
```

Comment:
remove yarn reference. npm 5 essentially is yarn. we do not reference yarn elsewhere nor plan to.